tests/framework/ensembleModelTests/test_reuse_em.xml

```diff
@@ -0,0 +1,188 @@
+    <batchSize>1</batchSize>
+    <internalParallel>True</internalParallel>
+  </RunInfo>
+
+  <Distributions>
+    <Uniform name="distA">
+      <lowerBound>0</lowerBound>
+      <upperBound>1</upperBound>
+    </Uniform>
+    <Uniform name="distB">
+      <lowerBound>2</lowerBound>
+      <upperBound>3</upperBound>
+    </Uniform>
+    <Uniform name="distH">
+      <lowerBound>5</lowerBound>
+      <upperBound>8</upperBound>
+    </Uniform>
+  </Distributions>
+
+  <Models>
+    <ExternalModel ModuleToLoad="alpha" name="Alpha" subType="">
+      <variables>A,B,C</variables>
+    </ExternalModel>
+    <ExternalModel ModuleToLoad="beta" name="Beta" subType="">
+      <variables>C,D,E</variables>
+    </ExternalModel>
+    <ExternalModel ModuleToLoad="gamma" name="Gamma" subType="">
+      <variables>D,G</variables>
+    </ExternalModel>
+    <ExternalModel ModuleToLoad="delta" name="Delta" subType="">
+      <variables>F,E</variables>
+    </ExternalModel>
+    <ExternalModel ModuleToLoad="epsilon" name="Epsilon" subType="">
+      <variables>H,G</variables>
+    </ExternalModel>
+    <ExternalModel ModuleToLoad="rho" name="Rho" subType="">
+      <variables>D, I</variables>
+    </ExternalModel>
+
+    <EnsembleModel name="testConfig" subType="">
+      <Model class="Models" type="ExternalModel">
+        Alpha
+        <Input   class="DataObjects" type="PointSet"      >inputAlpha</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataAlpha</TargetEvaluation>
+      </Model>
+      <Model class="Models" type="ExternalModel">
+        Beta
+        <Input   class="DataObjects" type="PointSet"      >inputBeta</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataBeta</TargetEvaluation>
+      </Model>
+      <Model class="Models" type="ExternalModel">
+        Delta
+        <Input   class="DataObjects" type="PointSet"      >inputDelta</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataDelta</TargetEvaluation>
+      </Model>
+      <Model class="Models"  type="ExternalModel">
+        Gamma
+        <Input   class="DataObjects" type="PointSet"      >inputGamma</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataGamma</TargetEvaluation>
+      </Model>
+      <Model class="Models"  type="ExternalModel">
+        Epsilon
+        <Input   class="DataObjects" type="PointSet"      >inputEpsilon</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataEpsilon</TargetEvaluation>
+      </Model>
+      <Model class="Models"  type="ExternalModel">
+        Rho
+        <Input   class="DataObjects" type="PointSet"      >inputRho</Input>
+        <TargetEvaluation class="DataObjects" type="PointSet">dataRho</TargetEvaluation>
+      </Model>
+    </EnsembleModel>
+  </Models>
+
+  <Samplers>
+    <MonteCarlo name="MC1">
+      <samplerInit>
+        <limit>6</limit>
+        <initialSeed>200286</initialSeed>
```

Comment:
This is redundant with the step reseeding; I would remove this one for clarity.

Reply:
removed